### PR TITLE
[Stratconn-2879] | Set up DataDog dashboard to track requests to Oauth Supported Destinations.

### PR DESCRIPTION
**Pull request by Innovative-GauravKochar**
This Pull request is to set up DataDog dashboard to track requests to Oauth Supported Destinations.
Jira Ticket:- https://segment.atlassian.net/jira/software/c/projects/STRATCONN/boards/310?modal=detail&selectedIssue=STRATCONN-2879&assignee=63617339fc0cc7a600b03c6b

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality - **Not Required**
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) - **Not Required**
- [ ] [Segmenters] Tested in the staging environment -

### CODE REVIEW

**Review by varadarajan-tw — APPROVED** [review]

File: packages/destination-actions/src/destinations/airship/utilities.ts [hunk 1]
```diff
@@ -229,8 +229,8 @@ function _build_attribute(attribute_key: string, attribute_value: any, occurred:
     adjustedDate = _parse_date(attribute_value)
   }
 
-  if (['home_phone', 'work_phone', 'mobile_phone'].includes(attribute_key) && typeof(attribute_value) == "string") {
```
[the remark this answers is not in the document]
Comment (Innovative-GauravKochar) [reply packages/destination-actions/src/destinations/airship/utilities.ts:232]: these changes are getting automatically added while committing the changes

**Comment by brennan** [comment]
@Innovative-GauravKochar Thank you for putting all this together! After seeing how many changes this entails, I think I'd prefer to extract LinkedIn into its own PR and merge/deploy that first so we limit the blast radius of the change. Once we get signal on how useful metrics are for LinkedIn, then we can move forward with adding metrics to the other destinations.

@Innovative-GauravKochar is it possible to set up a PR with only the changes to LinkedIn sometime this week? Thank you again!

**Comment by Innovative-GauravKochar** [comment]
@brennan  Created a Seperate PR for linkedin-audiences https://github.com/segmentio/action-destinations/pull/1619

**Comment by varadarajan-tw** [comment]
should we close this in favor of https://github.com/segmentio/integrations/pull/2920?

**Comment by Innovative-GauravKochar** [comment]
> Member

**Comment by Innovative-GauravKochar** [comment]
Closing in favour of https://github.com/segmentio/integrations/pull/2920

**Pull request closed without merging**